### ISSUE TITLE
Add step type for destination set filtering in the source transformer

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3706]
This would be useful to have so that users don't have to write custom JavaScript to do this.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3823
Reporter: narupley
Created: 2015-10-21T13:36:14.000-0700

**Comment by rbeckman-nextgen** [comment]
Revision 8217: A new source transformer step called Destination Set Filter has been added. This allows easier use of the destination set filtering feature we added in 3.1. Users can select the behavior (remove, removeAllExcept, removeAll), the specific destinations to exclude, and the actual conditional logic, which is the same format as the Rule Builder.


Imported Comment. Original Details:
Author: narupley
Created: 2017-03-22T11:38:38.000-0700
